### PR TITLE
Add corrections from `typos` dictionary (A1)

### PR DESCRIPTION
**Pull request by skangas**
These corrections were made available under an Apache/MIT dual license.

https://github.com/crate-ci/typos/blob/master/crates/typos-dict/assets/words.csv

### CODE REVIEW

**Comment by skangas** [comment]
> @skangas Could you rebase to (i) resolve conflicts and (ii) restart CI checks?

Done.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -123,6 +124,8 @@ abbrviating->abbreviating
 abbrviation->abbreviation
 abbrviations->abbreviations
 abcense->absence
+abck->back
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:127]:
```suggestion
abck->back, aback,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -213,7 +221,9 @@ abosulute->absolute
 abosulutely->absolutely
 abot->about, abort, a bot,
 abotu->about
+abou->about, abound,
 abount->about
+abour->about
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:226]:
```suggestion
abour->about, labour,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -2064,17 +2114,24 @@ adjsuts->adjusts
...
 adjustmant->adjustment
 adjustmants->adjustments
 adjustmenet->adjustment
+adknowledged->acknowledged
+adknowledges->acknowledges
 admendment->amendment
 admi->admin, admit, admix,
+adminastrator->administrator
+adminastrators->administrators
+adming->admin
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2134]:
```suggestion
adming->admin, arming,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -2153,17 +2236,39 @@ admissable->admissible
...
+admrial->admiral
+admrials->admirals
 adn->and
+adnimistrator->administrator
+adnimistrators->administrators
 adnroid->android
 adnroids->androids
 adobted->adopted
 adolecent->adolescent
+adolence->adolescent
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2256]:
```suggestion
adolence->adolescence
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -2246,39 +2371,96 @@ adventageous->advantageous
...
+advsie->advise
+advsied->advised
+advsior->advisor
+advsiors->advisors
 adwances->advances
 aeactivate->deactivate, activate,
+aeorspace->aerospace
 aequidistant->equidistant
 aequivalent->equivalent
+aer->are
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2444]: AER may stand for _annual equivalent rate_. Move to _rare_?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -2313,23 +2505,65 @@ affinties->affinities
...
+afghanastan->Afghanistan
+afghanisthan->Afghanistan
+afghansitan->Afghanistan
+afhganistan->Afghanistan
 afile->a file, agile,
 afinity->affinity
+afircan->African
+afircans->Africans
 afor->for
+aford->afford
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2543]:
```suggestion
aford->afford, afore,
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -2433,6 +2680,8 @@ aggregater->aggregator, aggregated, aggregates, aggregate,
 aggregaters->aggregators, aggregates,
 aggregatet->aggregated
 aggregatin->aggregating, aggregation,
+aggregatore->aggregate
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2683]:
```suggestion
aggregatore->aggregator
```
or
```suggestion
aggregatore->aggregator, aggregate,
```

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -2458,8 +2707,14 @@ aggregration->aggregation
 aggregrations->aggregations
 aggregrator->aggregator
 aggregrators->aggregators
+aggrement->agreement
+aggresions->aggression
 aggresive->aggressive
 aggresively->aggressively
+aggressivley->aggressively
+aggressivly->aggressively
+aggressivo->aggression
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2716]:
```suggestion
aggressivo->aggressive, aggression,
```

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -2524,36 +2801,66 @@ agregation->aggregation
...
 agrements->agreements
+agrentina->Argentina
+agressie->aggressive
 agression->aggression
 agressions->aggressions
 agressive->aggressive
 agressively->aggressively
 agressiveness->aggressiveness
 agressivity->aggressivity
+agressivley->aggressive
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2816]:
```suggestion
agressivley->aggressively
```

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -2524,36 +2801,66 @@ agregation->aggregation
...
+agrumentative->argumentative
 agruments->arguments
 ags->tags, ages,
 agsinst->against
 agument->argument, augment,
 agumented->augmented
 agumenting->augmenting
 aguments->arguments, augments,
+agurement->argument
+ahd->had
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2863]:
```suggestion
ahd->had, and,
```

**Comment by skangas** [comment]
Thanks for reviewing, I've incorporated all your suggestions.

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -2650,28 +2981,52 @@ akumulation->accumulation
...
 alchoholic->alcoholic
 alchol->alcohol
 alcholic->alcoholic
+alchool->alcohol
+alchoolic->alcoholic
+alchoolism->alcoholism
 alcohal->alcohol
+alcohalics->alcoholics
+alcohalism->alcoholism
+alcoholc->alcoholics
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3005]:
```suggestion
alcoholc->alcoholic
```

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -2650,28 +2981,52 @@ akumulation->accumulation
...
+alchemsit->alchemist
+alchimest->alchemist
+alchmeist->alchemist
+alchmey->alchemy
 alchohol->alcohol
 alchoholic->alcoholic
 alchol->alcohol
 alcholic->alcoholic
+alchool->alcohol
+alchoolic->alcoholic
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3000]:
```suggestion
alchoolic->alcoholic
alchoolics->alcoholics
```

File: codespell_lib/data/dictionary.txt [hunk 13]
```diff
@@ -2650,28 +2981,52 @@ akumulation->accumulation
...
+alchoolism->alcoholism
 alcohal->alcohol
+alcohalics->alcoholics
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3003]:
```suggestion
alcohalic->alcoholic
alcohalics->alcoholics
```

File: codespell_lib/data/dictionary.txt [hunk 14]
```diff
@@ -2650,28 +2981,52 @@ akumulation->accumulation
...
+alcohalism->alcoholism
+alcoholc->alcoholics
 alcoholical->alcoholic
+alcoholicas->alcoholics
+alcoholicos->alcoholics
+alcoholis->alcoholics
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3009]:
```suggestion
alcoholis->alcoholics, alcoholic,
```

File: codespell_lib/data/dictionary.txt [hunk 15]
```diff
@@ -2650,28 +2981,52 @@ akumulation->accumulation
...
+alcoholsim->alcoholism
+aldutery->adultery
 aleady->already
 aleays->always
+alechmist->alchemist
 aledge->allege
 aledged->alleged
 aledges->alleges
 alegance->allegiance
+alegbra->algebra
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3020]:
```suggestion
alegbra->algebra
alegbraic->algebraic
```

File: codespell_lib/data/dictionary.txt [hunk 16]
```diff
@@ -2650,28 +2981,52 @@ akumulation->accumulation
...
 alege->allege
 aleged->alleged
 alegiance->allegiance
 alegiances->allegiances
 alegience->allegiance
 alegorical->allegorical
+aleinated->alienated
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3027]:
```suggestion
aleinate->alienate
aleinated->alienated
aleinates->alienates
```

File: codespell_lib/data/dictionary.txt [hunk 17]
```diff
@@ -2650,28 +2981,52 @@ akumulation->accumulation
...
+aleinating->alienating
+aleniate->alienate
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3029]:
```suggestion
aleniate->alienate
aleniated->alienated
aleniates->alienates
aleniating->alienating
```

File: codespell_lib/data/dictionary.txt [hunk 18]
```diff
@@ -2691,9 +3046,11 @@ alforithmic->algorithmic
 alforithmically->algorithmically
 alforithms->algorithms
 algebraical->algebraic
+algebriac->algebraic
 algebric->algebraic
 algebrra->algebra
 algee->algae
+algerba->algebra
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3053]:
```suggestion
algerba->algebra
algerbaic->algebraic
```

File: codespell_lib/data/dictionary.txt [hunk 19]
```diff
@@ -2791,14 +3148,20 @@ algorithmn->algorithm
 algorithmnic->algorithmic
 algorithmnically->algorithmically
 algorithmns->algorithms
+algorithmus->algorithms
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3151]:
```suggestion
algorithmo->algorithm
algorithmos->algorithm, algorithms,
algorithmus->algorithm, algorithms,
```

File: codespell_lib/data/dictionary.txt [hunk 20]
```diff
@@ -2791,14 +3148,20 @@ algorithmn->algorithm
...
 algoriths->algorithms
 algorithsm->algorithm, algorithms,
 algorithsmic->algorithmic
 algorithsmically->algorithmically
 algorithsms->algorithms
+algorithum->algorithm
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3157]:
```suggestion
algorithum->algorithm
algorithums->algorithms
```

File: codespell_lib/data/dictionary.txt [hunk 21]
```diff
@@ -2791,14 +3148,20 @@ algorithmn->algorithm
...
+algorithym->algorithm
+algorithyms->algorithm
 algoritm->algorithm
+algoritmes->algorithms
 algoritmic->algorithmic
 algoritmically->algorithmically
+algoritmos->algorithms
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3164]:
```suggestion
algoritmos->algorithms, algorithm,
```

File: codespell_lib/data/dictionary.txt [hunk 22]
```diff
@@ -2872,6 +3235,7 @@ algorythem->algorithm
 algorythemic->algorithmic
 algorythemically->algorithmically
 algorythems->algorithms
+algorythims->algorithms
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3238]:
```suggestion
algorythim->algorithm
algorythims->algorithms
```

File: codespell_lib/data/dictionary.txt [hunk 23]
```diff
@@ -2900,6 +3264,7 @@ algotrithm->algorithm
 algotrithmic->algorithmic
 algotrithmically->algorithmically
 algotrithms->algorithms
+algrithm->algorithm
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3267]:
```suggestion
algrithm->algorithm
algrithms->algorithms
```

File: codespell_lib/data/dictionary.txt [hunk 24]
```diff
@@ -2975,6 +3343,8 @@ alignmnet->alignment
 alignmnt->alignment
 alignrigh->alignright
 alikes->alike, likes,
+alimoney->alimony
+alimunium->aluminium
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3347]: Add US spelling:
```suggestion
alimunium->aluminium
alimunum->aluminum
```

File: codespell_lib/data/dictionary.txt [hunk 25]
```diff
@@ -2993,6 +3364,9 @@ alised->aliased
 alises->aliases
 alising->aliasing
 aliver->alive, liver, a liver, sliver,
+allacritty->alacritty
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3367]:
```suggestion
allacritty->alacrity
allacrity->alacrity
```

Also add:
```
alacritty->alacrity
```

File: codespell_lib/data/dictionary.txt [hunk 26]
```diff
@@ -3017,21 +3391,37 @@ alled->called, allied,
...
+allegeance->allegiance
 allegedely->allegedly
+allegedley->allegedly
 allegedy->allegedly
 allegely->allegedly
 allegence->allegiance
+allegiancies->allegiance
 allegience->allegiance
 allegiences->allegiances
+allegric->allergic
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3407]:
```suggestion
allegric->allergic
allegries->allergies
```

File: codespell_lib/data/dictionary.txt [hunk 27]
```diff
@@ -3017,21 +3391,37 @@ alled->called, allied,
...
+allegry->allergy
+alleigance->allegiance
+alleigances->allegiance
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3410]:
```suggestion
alleigances->allegiances
```
Add singular `allegiance`?

File: codespell_lib/data/dictionary.txt [hunk 28]
```diff
@@ -3017,21 +3391,37 @@ alled->called, allied,
...
 alledges->alleges
 allegaince->allegiance
 allegainces->allegiances
+allegeance->allegiance
 allegedely->allegedly
+allegedley->allegedly
 allegedy->allegedly
 allegely->allegedly
 allegence->allegiance
+allegiancies->allegiance
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3404]:
```suggestion
allegiancies->allegiances
```
Add singular `allegiance`?

**Review by DimitriPapadopoulos — COMMENTED** [review]
A few more suggestions.

Generally speaking, we tend to add singular/plural and all verbal forms.

**Comment by skangas** [comment]
> A few more suggestions.

Thanks, incorporated.

> Generally speaking, we tend to add singular/plural and all verbal forms.

Indeed. I've spent a lot of time adding those to the original list.

File: codespell_lib/data/dictionary.txt [hunk 29]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
 adventages->advantages
 adventagous->advantageous
 adventagously->advantageously
+adventerous->adventurous
+adventourus->adventurous
 adventrous->adventurous
+adventrues->adventures
+adventue->adventures
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2378]:
```suggestion
adventue->adventure
```

File: codespell_lib/data/dictionary.txt [hunk 30]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
 adventages->advantages
 adventagous->advantageous
 adventagously->advantageously
+adventerous->adventurous
+adventourus->adventurous
 adventrous->adventurous
+adventrues->adventures
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2377]:
```suggestion
adventrue->adventure
adventrues->adventures
```

File: codespell_lib/data/dictionary.txt [hunk 31]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
+adventue->adventures
+adventuers->adventures, adventurers,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2379]:
```suggestion
adventuers->adventures, adventurers,
adventues->adventures
```

File: codespell_lib/data/dictionary.txt [hunk 32]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
+adventuous->adventurous
+adventureous->adventures
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2381]:
```suggestion
adventureous->adventurous
```

File: codespell_lib/data/dictionary.txt [hunk 33]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
+adverticement->advertisement
+advertis->adverts, advertise,
+advertisiers->advertisers
+advertisiment->advertisement
 advertisin->advertising
 advertisment->advertisement
 advertisments->advertisements
+advertisors->advertisers
+advertisted->advertised
+advertisters->advertisers
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2405]:
```suggestion
advertister->advertiser
advertisters->advertisers
```

File: codespell_lib/data/dictionary.txt [hunk 34]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
 advertice->advertise
 adverticed->advertised
+adverticement->advertisement
+advertis->adverts, advertise,
+advertisiers->advertisers
+advertisiment->advertisement
 advertisin->advertising
 advertisment->advertisement
 advertisments->advertisements
+advertisors->advertisers
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2403]:
```suggestion
advertisor->advertiser
advertisors->advertisers
```

File: codespell_lib/data/dictionary.txt [hunk 35]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
+adviseer->adviser
+adviseur->adviser
 advisin->advising
 advisoriy->advisory, advisories,
 advisoriyes->advisories
+advisorys->advisors, advisories,
 advizable->advisable
+advnace->advance
+advocade->advocated
+advocats->advocates
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2433]:
```suggestion
advocat->advocate
advocats->advocates
```

File: codespell_lib/data/dictionary.txt [hunk 36]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
 adviseable->advisable
+adviseer->adviser
+adviseur->adviser
 advisin->advising
 advisoriy->advisory, advisories,
 advisoriyes->advisories
+advisorys->advisors, advisories,
 advizable->advisable
+advnace->advance
+advocade->advocated
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2432]:
```
advocade->advocated, advocate,
```

File: codespell_lib/data/dictionary.txt [hunk 37]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
 aeriel->aerial
 aeriels->aerials
+aeropsace->aerospace
+aerosapce->aerospace
+aersopace->aerospace
 aesily->easily
+aestethic->aesthetic
+aestethically->aesthetically
+aestethics->aesthetics
+aesthatically->aesthetically
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2453]:
```suggestion
aesthatic->aesthetic
aesthatically->aesthetically
```

File: codespell_lib/data/dictionary.txt [hunk 38]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
+aesthatics->aesthetics
+aesthestic->aesthetics
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2455]:
```suggestion
aesthestic->aesthetics
aesthestically->aesthetically
```

File: codespell_lib/data/dictionary.txt [hunk 39]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
+aesthethics->aesthetics
+aestheticaly->aesthetically
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2457]:
```suggestion
aesthetic->aesthetic
aestheticaly->aesthetically
```

File: codespell_lib/data/dictionary.txt [hunk 40]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
+aestheticlly->aesthetically
+aesthitically->aesthetically
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2459]:
```suggestion
aesthitically->aesthetically
aesthitics->aesthetics
```

File: codespell_lib/data/dictionary.txt [hunk 41]
```diff
@@ -2246,39 +2371,95 @@ adventageous->advantageous
...
 aesy->easy
+aethistic->atheistic
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2461]:
```suggestion
aethist->atheist
aethistic->atheistic
```

File: codespell_lib/data/dictionary.txt [hunk 42]
```diff
@@ -2290,12 +2471,19 @@ afected->affected
 afecting->affecting
 afects->affects, effects,
 afer->after
+afernoon->afternoon
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2474]:
```suggestion
afernoon->afternoon
afernoons->afternoons
```

File: codespell_lib/data/dictionary.txt [hunk 43]
```diff
@@ -2290,12 +2471,19 @@ afected->affected
...
 aferwards->afterwards
+afeter->after
 afetr->after
 affact->affect, effect,
+affaires->affairs
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2479]:
```suggestion
affaire->affair
affaires->affairs
```

File: codespell_lib/data/dictionary.txt [hunk 44]
```diff
@@ -2313,23 +2504,65 @@ affinties->affinities
 affintiy->affinity
 affintize->affinitize
 affinty->affinity
+affirmate->affirmative
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2507]:
```suggestion
affirmate->affirm, affirmative,
affirmates->affirms
```

File: codespell_lib/data/dictionary.txt [hunk 45]
```diff
@@ -2313,23 +2504,65 @@ affinties->affinities
...
 aforememtioned->aforementioned
+aforementioend->aforementioned
 aforementiond->aforementioned
 aforementionned->aforementioned
 aformentioned->aforementioned
 afrer->after
+afriad->afraid
+africain->African
+africanas->Africans
+africaners->Africans
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2552]:
```suggestion
africaner->African, Afrikaner, Africander, Afrikander,
africaners->Africans, Afrikaners, Africanders, Afrikanders,
```

File: codespell_lib/data/dictionary.txt [hunk 46]
```diff
@@ -2313,23 +2504,65 @@ affinties->affinities
...
 afor->for
+aford->afford, afore,
 aforememtioned->aforementioned
+aforementioend->aforementioned
 aforementiond->aforementioned
 aforementionned->aforementioned
 aformentioned->aforementioned
 afrer->after
+afriad->afraid
+africain->African
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2550]:
```suggestion
africain->African
africains->Africans
```

File: codespell_lib/data/dictionary.txt [hunk 47]
```diff
@@ -2313,23 +2504,65 @@ affinties->affinities
...
+africanas->Africans
+africaners->Africans
+africaness->Africans
+africanos->Africans
+afte->after
 afterall->after all
+afterhtought->afterthought
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2557]:
```suggestion
afterhtought->afterthought
afterhtoughts->afterthoughts
```

File: codespell_lib/data/dictionary.txt [hunk 48]
```diff
@@ -2562,34 +2868,60 @@ ahlpa->alpha
 ahlpas->alphas
 ahmond->almond
 ahmonds->almonds
+ahould->should, ahold,
 ahppen->happen
+ahppy->happy
+ahtiest->atheist
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2874]:
```suggestion
ahtiest->atheist
ahtiests->atheists
```

File: codespell_lib/data/dictionary.txt [hunk 49]
```diff
@@ -2650,28 +2985,61 @@ akumulation->accumulation
...
+alchoolism->alcoholism
 alcohal->alcohol
+alcohalic->alcoholic
+alcohalics->alcoholics
+alcohalism->alcoholism
+alcoholc->alcoholic
 alcoholical->alcoholic
+alcoholicas->alcoholics
+alcoholicos->alcoholics
+alcoholis->alcoholics, alcoholic,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:3016]:
```suggestion
alcoholis->alcoholics, alcoholic, alcoholism,
```

**Review by DimitriPapadopoulos — COMMENTED** [review]
Sorry for the multiple rounds, these should be my last comments!

**Comment by DimitriPapadopoulos** [comment]
Thank you @skangas!

**Pull request merged**